config/environment.rb

```diff
@@ -1,5 +1,15 @@
 # Load the Rails application.
 require_relative 'application'
 
+ActionMailer::Base.smtp_settings = {
+  :user_name => ENV['SENDGRID_USERNAME'],
+  :password => ENV['SENDGRID_PASSWORD'],
+  :domain => 'yourdomain.com',
```

Comment:
Since this is separately specified in the development and production environments, can this line be removed here?

Reply:
Ok